src/xtgeo/cube/cube1.py

```diff
@@ -128,20 +253,29 @@ def __init__(
         self._yinc = yinc
         self._zinc = zinc
         self._yflip = yflip
-        self._zflip = 1  # currently not in use
+        self._zflip = zflip  # currently not in use
         self._rotation = rotation
+
+        self._values = None
         self.values = values  # "values" is intentional over "_values"; cf. values()
-        self._ilines = np.array(range(1, self._ncol + 1), dtype=np.int32)
-        self._xlines = np.array(range(1, self._nrow + 1), dtype=np.int32)
-        self._traceidcodes = np.ones((self._ncol, self._nrow), dtype=np.int32)
-        self._segyfile = None
-        self._metadata = xtgeo.MetaDataRegularCube()
-        self._undef = xtgeo.UNDEF
+
+        if ilines is None:
```

Comment:
Nit: perhaps  `self._ilines = ilines or np.array(range(1, self._ncol + 1), dtype=np.int32)`

Reply:
Fixed

Reply:
It's an alternative, but I don't know if an inline reads any better in this case.